_posts/2021-03-03-emb-comp.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Study: Comparing high-dimensional embeddings"
+date: 2021-03-03 12:00:00 -0400
+studyLink: "https://docs.google.com/forms/d/e/1FAIpQLSf6UEl4LaiCMdUvsnnCCM7l9xb2zx6MtWmBdrNg7WzjcknpXA/viewform?usp=sf_link"
+excerpt_separator: <!--more-->
+side_image: "/assets/posts/emb-comp-study.png"
+side_image_alt: "A series of example embedding visualizations."
+---
+
+We are building a tool for people who work with high-dimensional embedding spaces, a type of data common to many fields such as natural language processing, computer vision, and computational biology. In order to establish how embedding users understand and use their data, we are interviewing embedding model experts across these domains - for anyone who has worked extensively with embedding models in a recent project, we would love to <a href="{{ page.studyLink }}" target="_blank">hear about your experiences</a>!
+
+<!--more-->
```

Comment:
Doe jekyll's automatic split not work well enough?

Reply:
It does work, I just added this as an example for future reference on how to do it